source/PluginDev/Assets/GooglePlayGames/ISocialPlatform/PlayGamesLocalUser.cs

```diff
@@ -132,6 +132,20 @@ public new string id
         }
 
         /// <summary>
+        /// Gets an access token for the user.
+        /// </summary>
+        /// <returns>
+        /// An access token for the user.
+        /// </returns>
+        public new string token
+        {
+            get
+            {
+                return authenticated ? mPlatform.GetToken() : string.Empty;
```

Comment:
should return null?  this way it is consistent with PlayGamesPlatform.GetToken()